src/Pact/Interpreter.hs

```diff
@@ -216,13 +218,21 @@ setupEvalEnv dbEnv ent mode msgData refStore gasEnv np spv pd ec = do
         toPair Signer{..} = (pk,S.fromList _siCapList)
           where
             pk = PublicKeyText $ fromMaybe _siPubKey _siAddress
+    mkVerifiers m = M.fromListWith S.union $ map toPair m
```

Comment:
This union to map will silently swallow any pairs (args, caplist) that differ on the caplist but agree on the args. Is that what we want?

Reply:
Pact doesn't care about args at all, just about verifier names and caps being associated.